script.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+mkdir task
```

Comment:
При использовании флага `-p` можно сразу так делать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```